javascript/form.js

```diff
@@ -0,0 +1,144 @@
+const form = document.querySelector('[data-js="form"]');
+const main = document.querySelector('[data-js="form-main"]');
+let idCounter = 0;
+
+const listOfTextareas = document.querySelectorAll('[data-js="form__textarea"]');
+
+listOfTextareas.forEach((singleTextarea) => {
+  singleTextarea.addEventListener("input", (event) => {
+    const numberOfCharactersEntered = event.target.value.length;
+    const maxCharacters = event.target.getAttribute("maxlength");
+    const charactersLeft = maxCharacters - numberOfCharactersEntered;
+    const counter = event.target.nextElementSibling.querySelector(
+      '[data-js="character-counter"]'
+    );
+    counter.textContent = charactersLeft;
+  });
+});
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formElements = event.target.elements;
+  const question = formElements.form__question.value;
+  const answer = formElements.form__answer.value;
+  const code = formElements.form__code.value;
+  const tag = formElements.form__tag.value;
```

Comment:
Hier habe ich vorhin recherchiert. Man kann mit `.split(" ").join("")` die Leerzeichen entfernen, wie bei Hashtags üblich.

Reply:
Guter Tipp, danke. Werde ich mir mal bei mdn anschauen und vermutlich bei mir einfügen. .split("") hatten wir auch schon in irgendeiner Challenge mal. :grinning: :+1: